.goreleaser.yml

```diff
@@ -16,10 +18,9 @@ snapshot:
   name_template: "{{ .Tag }}-snapshot"
 
 dockers:
-
 - image_templates:
-  - "docker.io/vshn/k8up:v{{ .Version }}"
-  - "quay.io/vshn/k8up:v{{ .Version }}"
+  - "docker.io/vshn/k8up:{{ .Version }}"
+  - "quay.io/vshn/k8up:{{ .Version }}"
```

Comment:
The `.Version` comes without the `v` prefix, so we should keep it there.
https://goreleaser.com/customization/templates/

Reply:
Interesting. If I run goreleaser on my host, this is what I get:

```
   • docker images
      • building docker image     image=docker.io/vshn/wrestic:v1.2.0-snapshot
      • building docker image     image=docker.io/vshn/k8up:v1.2.0-snapshot
      • building docker image     image=docker.io/vshn/wrestic:v1.2.0-snapshot-nonroot
      • pipe skipped              error=publishing is disabled
```

But if I change it to `"docker.io/vshn/k8up:{{ .Version }}"`, then I get:

```
   • docker images
      • building docker image     image=docker.io/vshn/k8up:vv1.2.0-snapshot
      • building docker image     image=docker.io/vshn/wrestic:vv1.2.0-snapshot
      • building docker image     image=docker.io/vshn/wrestic:vv1.2.0-snapshot-nonroot
      • pipe skipped              error=publishing is disabled
```

That's why I dropped the 'v' in the first place.

Reply:
I've found the bug. I was running `goreleaser --snapshot` and in this mode the substitution of `vX.Y.Z` to `X.Y.Z` does not seem to work.